packages/commonwealth/client/scripts/views/components/component_kit/new_designs/CWBanner.tsx

```diff
@@ -41,6 +42,7 @@ interface CWBannerProps {
   buttons?: ButtonProps[];
   className?: string;
   onClose: () => void;
+  gatedGroups?: Array<{ id: number; name: string }>;
```

Comment:
can we make this a generic react node like `footer: ReactNode` and add the gated tags from wherever the component gets used?